src/store/plugins/pendingTransactionHandler.js

```diff
@@ -24,7 +24,6 @@ export default async (store) => {
           showSpendModal();
           break;
         case 'spendToken':
-          store.dispatch('fungibleTokens/loadTokenBalances');
```

Comment:
After this change in the worst case the token balance will be updated in 10 seconds (current polling time).

Reply:
replaced with https://github.com/aeternity/superhero-wallet/pull/1823/files#diff-72b4f519bcdf7b7f49059fbea39c36b6c760d84199e160db539b0879f115d8f3R126

the issue with that is, it loads the token balances instantly after the transaction has been sent and MDW needs a few seconds to sync updates.

So that line was not updating the balances properly